qCC/ccClippingBoxTool.cpp

```diff
@@ -22,6 +22,7 @@
 #include "ccClippingBoxRepeatDlg.h"
 #include "ccCropTool.h"
 #include "ccGLWindow.h"
+#include "ccReservedDs.h"
```

Comment:
was ccReservedDs.h intentional or should it be ccReservedIDs.h ?

Reply:
Oops! Nice catch!